core/comments/comment_view.ts

```diff
@@ -47,103 +47,148 @@ export class CommentView implements IRenderedElement {
       'class': 'blocklyComment blocklyEditable',
     });
 
-    this.topBar = dom.createSvgElement(
+    ({
+      topBar: this.topBar,
```

Comment:
Yesh! It is a little bit silly because if you try to reference the property in the creation method typescript doesn't realize you're doing a bad. (E.g. reference `this.textArea` in `createTextArea` throws a runtime error) But that's not specific to what I'm doing here, typescript just isn't smart enough to track references from the constructor.